.github/workflows/build-debian-multiarch.yml

```
@@ -77,7 +77,7 @@ jobs:
         install: |
           apt-get update --fix-missing
           apt-get upgrade -y
-          apt-get install libsdl2-dev libsdl2-image-dev libsdl2-mixer-dev libsdl2-ttf-dev libfreetype6-dev libportmidi-dev libjpeg-dev fontconfig -y
+          apt-get install build-essential libsdl2-dev libsdl2-image-dev libsdl2-mixer-dev libsdl2-ttf-dev libfreetype6-dev libportmidi-dev fontconfig -y
           apt-get install python3-setuptools python3-dev python3-pip python3-wheel python3-sphinx -y
```

A suggestion for readability...
```suggestion
          apt-get install build-essential libsdl2-dev libsdl2-image-dev libsdl2-mixer-dev \
                          libsdl2-ttf-dev libfreetype6-dev libportmidi-dev fontconfig python3-dev \
                         python3-pip python3-setuptools python3-sphinx python3-wheel -y
```

---

Good suggestion, but I vaguely remember trying something like this in the past and it not working for mysterious reasons